backend/segmentation/models/mamba_unet.py

```
@@ -0,0 +1,146 @@
+            self.decoder_blocks.append(
+                DoubleConv(features[i], features[i - 1], use_instance_norm, dropout=dropout_rate if i > 1 else 0.0)
+            )
+
+        # Final 1x1
+        self.final_conv = nn.Conv2d(features[0], out_channels, kernel_size=1)
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        x = self.init_conv(x)
+        skips = [x]
+        # Encoder loop excludes the last down (last down is fused with bottleneck)
+        for pool, down in zip(self.pools[:-1], self.downs[:-1]):
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)